docs/docs/contributor/server/basics/zapier.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Zapier App
+sidebar_position: 3
+sidebar_custom_props:
+  icon: TbBrandZapier
+---
+
+## Setup
+- Ask an admin to access the **Twenty 1Password** account
+- Install [Zapier CLI](https://platform.zapier.com/quickstart/cli-tutorial) globally: `npm install -g zapier-platform-cli`
+- Login with CLI: `zapier login`
+  > email: shared@twenty.com
+
+  > password: Find it in the **Vault TwentyShared**, look for Zapier credentials
+
+- Install Zapier packages:
+```
+cd packages/twenty-zapier
+yarn
+```
+- set environment variables:
+  - In **packages/twenty-zapier** launch `cp .env.example .env`
+  - [launch backend application](/contributor/server/basics/workflows#starting-a-new-feature)
+  and go to `http://localhost:3000/graphql`
+  - set **Headers** with `{"Authorization": "Bearer YOUR_TOKEN"}`.
+  Find your token by connecting to the
+  [local front application](/contributor/frontend/basics/contributing#starting-a-new-feature)
+  and find the token in **graphql** requests' header.
+  - generate an apiKey:
+  ```
```

Comment:
maybe replace this by the upcoming usage of the Settings > Developer page

Reply:
ok